frontend/src/bundles/overview/components/goal-widget/components/circle-progress/circle-progress.tsx

```diff
@@ -0,0 +1,39 @@
+type CircularProgressProperties = {
+    value: number;
+    target: number;
+};
+
+const CircularProgress = ({
+    value,
+    target,
+}: CircularProgressProperties): JSX.Element => {
+    const radius = 72;
+    const stroke = 10;
+    const normalizedRadius = radius - stroke * 2;
+    const circumference = normalizedRadius * 2 * Math.PI;
```

Comment:
I think you should move those variables into constants folder and a give a little bit clearer names so others can understand what they mean

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/5bc911fb-a8a8-4a15-87bb-85abb9ff607b)